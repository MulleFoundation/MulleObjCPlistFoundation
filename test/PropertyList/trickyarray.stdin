( NO, <0de20271 431d9847 f08a0365>, <0a2f0147 4aa504df f0941780>, <0a910147 4aa506ee f0931780>, 120.4, 120.4, __NULL__, 8, __NULL__, 0 )
