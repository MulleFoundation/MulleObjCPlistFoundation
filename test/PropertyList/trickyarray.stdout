(
    NO,
    <0de20271 431d9847 f08a0365>,
    <0a2f0147 4aa504df f0941780>,
    <0a910147 4aa506ee f0931780>,
    "120.4",
    "120.4",
    NSNull,
    8,
    NSNull,
    0
)
